cdap-ui/app/cdap/components/Replicator/Create/TopPanel/PluginInfo.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import { objectQuery } from 'services/helpers';
+
+const styles = (theme): StyleRules => {
+  return {
+    root: {
+      lineHeight: 1.2,
+    },
+    type: {
+      color: theme.palette.grey[200],
+    },
+    name: {
+      fontSize: '14px',
```

Comment:
nit: Doesn't this come from the html tag?

Reply:
no, we changed the html default to 13px